pkg/pprof/pprof.go

```diff
@@ -437,9 +394,20 @@ func (c *Converter) AddUnwinderInfoLocation(frameID uint64) *pprofprofile.Locati
 		return l
 	}
 
+	mapping := c.interpreterMapping
+	for idx, prog := range bpfprograms.ProgNames {
+		if base, found := strings.CutSuffix(interpreterSymbol.Filename, ".c"); found {
+			_, file := path.Split(base)
+			if strings.HasSuffix(prog, file+".o") {
+				mapping = c.bpfProgMappings[idx]
```

Comment:
This is kinda hacky, open to suggestions.

Reply:
We use `StackFrame.Status` in `profile.go` to track truncated frames today. You could add a new possible value for bpf error frames, then pass it through to here.